tests/conftest.py

```diff
@@ -3,45 +3,24 @@
 
 @pytest.fixture
 def user_filesystem(tmp_path):
-    # create a filesystem with all expected files.
-    ck_path = tmp_path / ".cookiecutters"
-    ck_path.mkdir()
-    repo_path = ck_path / "scikit-package-manuscript"
-    repo_path.mkdir()
+    # create a filesystem with an empty ".cookiecutters" directory.
     cwd_dir = tmp_path / "cwd_dir"
     cwd_dir.mkdir()
+    ck_path = tmp_path / ".cookiecutters"
+    ck_path.mkdir()
 
-    manuscript_path = repo_path / "templates" / "article" / "manuscript.tex"
-    manuscript_path.parent.mkdir(parents=True, exist_ok=True)
-    manuscript_path.touch()
+    yield tmp_path, ck_path, cwd_dir
 
-    yield tmp_path, cwd_dir
 
 
 @pytest.fixture
-def user_filesystem_without_repo(tmp_path):
-    # Create a filesystem without the "scikit-package-manuscript" repo cloned.
-    ck_path = tmp_path / ".cookiecutters"
-    ck_path.mkdir()
-    repo_path = ck_path / "foo"
-    repo_path.mkdir()
+def user_filesystem_with_repo(tmp_path):
+    # create a filesystem with an empty ".cookiecutters" directory.
     cwd_dir = tmp_path / "cwd_dir"
     cwd_dir.mkdir()
-    yield tmp_path, cwd_dir
-
-
-@pytest.fixture
-def user_filesystem_without_template(tmp_path):
-    # Create a filesystem without the "article" template in the "scikit-package-manuscript" repo.
     ck_path = tmp_path / ".cookiecutters"
     ck_path.mkdir()
-    repo_path = ck_path / "scikit-package-manuscript"
-    repo_path.mkdir()
-    cwd_dir = tmp_path / "cwd_dir"
-    cwd_dir.mkdir()
-
-    manuscript_path = repo_path / "templates" / "foo" / "manuscript.tex"
-    manuscript_path.parent.mkdir(parents=True, exist_ok=True)
-    manuscript_path.touch()
+    skm_path = ck_path / "scikit-package-manuscript"
```

Comment:
There seems to be some redundancy here.  I wonder whether we can just build the whole file-structure and then yield `tmp_dir` and everything below it will be available in the tests?